docs/releases/node-releases.md

```diff
@@ -151,7 +151,7 @@ As part of the original implementation work we were able to setup infrastructure
 
 * **BEHAVIOR CHANGE** [`process`](/commands/rpc-protocol/#process) now takes an optional flag `watch_work` (default `true`). Unless set to `false`, processed blocks can be subject to PoW rework
 * **BEHAVIOR CHANGE** [`bootstrap`](/commands/rpc-protocol/#bootstrap), [`bootstrap_any`](/commands/rpc-protocol/#bootstrap_any) and [`boostrap_lazy`](/commands/rpc-protocol/#bootstrap_lazy) will now throw errors when certain launch flags are used to disabled bootstrap methods - see each RPC page for details
-* **BEHAVIOR CHANGE** RPCs requiring work generation will now throw errors when work generation is disabled (no [work peers](/running-a-node/configuration/#nodework_peers), no [OpenCL](/running-a-node/configuration/#openclenable) and no work threads configured)
+* **BEHAVIOR CHANGE** RPCs requiring work generation will now throw errors when work generation is disabled (no [work peers](/integration-guides/work-generation/#nodework_peers), no [OpenCL](/integration-guides/work-generation/#nodeopenclenable) and no work threads configured)
```

Comment:
This fix was already moved to the new "Previous Release Notes" page here https://github.com/nanocurrency/nano-docs/pull/271/commits/525e3dfc31fa2b40c4b91d0cd33a3c8e382730ca